lms/util/jwt.py

```diff
@@ -1,11 +1,19 @@
 import jwt
 
+from pyramid.i18n import TranslationString as _
+
+from lms.exceptions import MissingLTILaunchParamError
 
 def build_jwt_from_lti_launch(lti_params, jwt_secret):
     """Build a client jwt token from lti_launch params."""
+    try:
+        roles = lti_params['roles']
+    except KeyError:
+        raise MissingLTILaunchParamError(_('LTI data parameter roles is required for launch.'))
```

Comment:
A thought...I'm seeing this Exception used in a bunch of places (which is cool) and it always uses the formula `"LTI data parameter {param_name} is required for launch"` for its message. I wonder if the Exception constructor could take a role name parameter and the Exception could build its own default message so you don't have to repeat code? Would prevent you from having to import `i18n` any time you want to use it...anyway. a thought...

Reply:
Sure! But since there are a few other PRs that are pending review(and some merged), could we save it as a followup PR?